src/main/java/ti4/commands/uncategorized/PlanetsInfo.java

```diff
@@ -0,0 +1,45 @@
+package ti4.commands.uncategorized;
+
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import net.dv8tion.jda.api.requests.restaction.CommandListUpdateAction;
+import ti4.commands.Command;
+import ti4.generator.Mapper;
+import ti4.helpers.Constants;
+import ti4.map.GameManager;
+import ti4.message.MessageHelper;
+
+public class PlanetsInfo implements Command, InfoThreadCommand {
```

Comment:
This command should go under the PlanetComment as a subcommand, so `/planet info` instead of `/planets_info`